src/components/QuranReader/LearningPlanBanner/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import Image from 'next/image';
+import useTranslation from 'next-translate/useTranslation';
+
+import styles from './LearningPlanBanner.module.scss';
+
+import Button, { ButtonSize } from '@/dls/Button/Button';
+import Link from '@/dls/Link/Link';
+import { logButtonClick } from '@/utils/eventLogger';
+import { getLearningPlanBannerUrl } from '@/utils/quranReflect/navigation';
+
+const LEARNING_PLAN_SLUG = 'the-rescuer-powerful-lessons-in-surah-al-mulk';
+const LEARNING_PLAN_URL = getLearningPlanBannerUrl(LEARNING_PLAN_SLUG);
+const BANNER_IMAGE_PATH =
+  'https://images.quran.com/the-rescuer-powerful-lessons-in-surah-al-mulk/Banner.png';
+const BANNER_WIDTH = 1230;
+const BANNER_HEIGHT = 307;
+
+const LearningPlanBanner: React.FC = () => {
+  const { t } = useTranslation('quran-reader');
+
+  return (
+    <aside
+      className={styles.bannerWrapper}
+      aria-label={t('learning-plan-banner.banner-wrapper-aria-label')}
+      aria-live="polite"
+    >
+      <div className={styles.bannerContainer}>
+        <p className={styles.mobileCaption}>
+          <strong className={styles.captionBold}>{t('learning-plan-banner.main-headline')} </strong>
+          {t('learning-plan-banner.subtitle-description')}
+        </p>
+
+        <Link
+          href={LEARNING_PLAN_URL}
+          ariaLabel={t('learning-plan-banner.banner-image-description')}
+          onClick={() => logButtonClick('learning_plan_banner_image')}
+        >
+          <div className={styles.imageWrap}>
+            <Image
+              src={BANNER_IMAGE_PATH}
+              alt={t('learning-plan-banner.banner-image-alt-text')}
+              width={BANNER_WIDTH}
+              height={BANNER_HEIGHT}
+            />
+          </div>
+        </Link>
+
+        <Button
+          size={ButtonSize.Medium}
+          href={LEARNING_PLAN_URL}
+          className={styles.mobileButton}
+          ariaLabel={t('learning-plan-banner.button-accessibility-label')}
+          onClick={() => logButtonClick('learning_plan_banner_cta')}
+        >
+          {t('learning-plan-banner.call-to-action-button')}
+        </Button>
+
+        <div className={styles.desktopCaptionRow}>
+          <p className={styles.desktopCaption}>
+            <strong className={styles.captionBold}>
+              {t('learning-plan-banner.main-headline')}{' '}
+            </strong>
+            {t('learning-plan-banner.subtitle-description')}
+          </p>
+          <Button
+            size={ButtonSize.Medium}
+            href={LEARNING_PLAN_URL}
+            className={styles.desktopButton}
+            ariaLabel={t('learning-plan-banner.button-accessibility-label')}
+            onClick={() => logButtonClick('learning_plan_banner_cta')}
+          >
+            {t('learning-plan-banner.call-to-action-button')}
+          </Button>
```

Comment:
@yousefdergham I just noticed the duplication of components between mobile and desktop. We should not do this as it violated the DRY concept as the code is almost the same except for the css. We should be able to achieve the same UI via scss styling targetting different devices inside the `scss` file.